pallets/creditcoin/src/lib.rs

```diff
@@ -1249,6 +1285,61 @@ pub mod pallet {
 			Ok(())
 		}
 
+		#[transactional]
+		#[pallet::weight(<T as Config>::WeightInfo::register_transfer_ocw())]
+		pub fn register_funding_transfer(
```

Comment:
I was expecting `Check Runtime and Extrinsics / Extrinsics` to fail here b/c the rename and duplcating of these extrinsics is changing the order in which they are defined.